### PR TITLE
Add Newo Bit and Cheat Manager

### PR DESCRIPTION
**Pull request by playerjmr**
- [x] Have you checked that there aren't other open pull requests for the same manifest update/change?
- [x] Have you verified that the manifest contains valid JSON?
- [x] Are you certain that this manifest is not designed to obtain copyrighted material or software, which you do not have permission to distribute?
- [x] Submitting malicious software is strictly forbidden, and potentially constitutes criminal activity, punishable by law. To the best of your knowledge, is this submission clear of any malicious intent?

Note: *filename*.oscmeta is the **slug** of the application you are submitting. This slug is used to locate files such as "/apps/**slug**/boot.dol" and "/apps/**slug**/meta.xml" in the final archive generated by this manifest. These files should be available at this location.

---

Newo Bit is a puzzle game created by [Owen](https://owensoft.net/project/newogame/) in which you place planets in orbit.
Cheat Manager is a utility that allows you to write, edit, and manage cheat code text files and even convert them to a usable GCT format for your games directly from the Wii console.

### CODE REVIEW

File: contents/cheat-manager.oscmeta [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        "peripherals": [
+            "Wii Remote",
+            "SDHC"
+        ],
+        "version": "0.3"
+    },
+    "source": {
+        "type": "url",
+        "format": "rar",
+        "location": "https://wiibrew.org/w/images/c/c7/Cheat_manager_v0_3.rar",
```
Comment (GabubuAvailable) [inline contents/cheat-manager.oscmeta:20]: remove the comma from line 20

File: contents/cheat-manager.oscmeta [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    },
```
Comment (GabubuAvailable) [inline contents/cheat-manager.oscmeta:21]: remove the comma from line 21

File: contents/newobit.oscmeta [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            "Classic Controller",
+            "GameCube Controller",
+            "SDHC"
+        ],
+        "version": "1.0"
+    },
+    "source": {
+        "type": "url",
+        "format": "zip",
+        "location": "https://owensoft.net/project/newogame/dl/newobit.zip",
```
Comment (GabubuAvailable) [inline contents/newobit.oscmeta:23]: remove the comma from line 23

**Review by GabubuAvailable — CHANGES_REQUESTED** [review]
Please consider fixing them before merging. Thanks!

**Review by GabubuAvailable — APPROVED** [review]

**Comment by Artuto** [comment]
For the record, the manifest for Cheat Manager was wrong, folder names must be the same as the app slug.

**Pull request merged**